aggregator/hitbtc.go

```diff
@@ -32,7 +32,14 @@ func (b *hitbtc) FetchPrice(ctx context.Context, base, quote string) (float64, e
 	if hpr.Price == "" {
 		return 0, ErrInvalidResponseData
 	}
-	return StrToFloat64(hpr.Price)
+	price, err := StrToFloat64(hpr.Price)
+	if err != nil {
+		return 0, err
+	}
+	if price <= 0 {
```

Comment:
since this will be duplicated over all aggregators, I would move this check in the StrToFloat64 directly and use it just about everywhere. And I would call this: StrToPositiveFloat64 or something. 
I guess this can be done in a separate PR.

Reply:
noted, will fix it in the next PR with all the fetchers

Reply:
ok, next PR will be

Reply:
:+1: